content/en/docs/tasks/security/mutual-tls/index.md

```diff
@@ -103,9 +103,9 @@ The output shows:
 
 * `CLIENT`: the mode used on the client or clients.
 
-* `AUTHN POLICY`: the name and namespace of the authentication policy. If the policy is the mesh-wide policy, namespace is blank, as in this case: `default/`
+* `AUTHN POLICY`: the namespace and name of the authentication policy. If the policy is the mesh-wide policy, namespace is blank, as in this case: `/default`
```

Comment:
Why is the authentication policy syntax and destination rule syntax not consistent? AP is `namespace/x` while DR is `x/namespace`. That inconsistency seems unnecessary and confusing.

Reply:
My bad, forgot to update the value for destination rule. Both are namespace/name order.